server/internal/infrastructure/gcp/file.go

```diff
@@ -91,7 +95,8 @@ func (f *fileRepo) ReadAsset(ctx context.Context, u string, fn string, h map[str
 
 func (f *fileRepo) GetAssetFiles(ctx context.Context, u string) ([]gateway.FileEntry, error) {
 	p := getGCSObjectPath(u, "")
-	b, err := f.bucket(ctx)
+	// Use standard client for GetAssetFiles to avoid bucket name corruption
+	b, err := f.bucketWithOptions(ctx, true)
```

Comment:
If we use the standard client here, the usage will not be tracked by the gcsproxy